frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bc.ts

```diff
@@ -0,0 +1,128 @@
+  SPREAD = 'SPREAD',
+  MIRROR = 'MIRROR',
+}
+
+const TierCountMap = Object.freeze({
+  [Tier.SPREAD]: 1,
+  [Tier.MIRROR]: 2,
+});
+
+export const cleanup = () => {
+  cy.exec('oc delete secret test-secret-for-bs -n openshift-storage', { failOnNonZeroExit: false });
+  cy.exec('oc delete backingstore test1 test2 test3 test4 -n openshift-storage', { failOnNonZeroExit: false });
+  cy.exec('oc delete backingstore noobaa-default-backing-store -n openshift-storage', { failOnNonZeroExit: false });
+  cy.exec('oc delete bucketclass test-affwe-bucketclass -n openshift-storage', { failOnNonZeroExit: false });
+}
+
+export const mockDefaultBS = {
+  apiVersion: 'noobaa.io/v1alpha1',
+  kind: 'BackingStore',
+  metadata: {
+    name: 'noobaa-default-backing-store',
+  },
+  spec: {
+    awsS3: {
+      region: 'us-east-1',
+      secret: {
+        name: MOCK_SECRET,
+        namespace: 'openshift-storage',
+      },
+      targetBucket: 'ssd',
+    },
+    ssl: false,
+    type: 'aws-s3',
+  },
+}
+
+export const createDummyBackingStore = (name: string) => {
+  cy.exec(`oc create secret generic ${MOCK_SECRET} -n openshift-storage --from-literal=password='iamthewatcherofthewall'`, { failOnNonZeroExit: false });
+  cy.exec(`echo '${JSON.stringify(mockDefaultBS)}' | kubectl create -n openshift-storage -f -`, { failOnNonZeroExit: false });
+  mockDefaultBS.metadata = Object.assign(_.pick(mockDefaultBS.metadata, ['name']), { name });
+  cy.log(JSON.stringify(mockDefaultBS));
+  cy.exec(`echo '${JSON.stringify(mockDefaultBS)}' | kubectl create -n openshift-storage -f -`);
+};
+
+const tierLevelToButton = (level: number, tier: Tier) =>
+level === 1
+  ? tier === Tier.SPREAD
+    ? cy.byTestID('placement-policy-spread1')
+    : cy.byTestID('placement-policy-mirror1')
+  : tier === Tier.SPREAD
+    ? cy.byTestID('placement-policy-spread2')
+    : cy.byTestID('placement-policy-mirror2');
+
+export const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}`;
+  
+
+
+export class BucketClassHandler {
+  name: string;
+
+  namespace: string;
+
+  description: string;
+
+  tiers: Tier[];
+
+  constructor(name: string, description: string, namespace: string = 'openshift-storage') {
+    this.name = name;
+    this.namespace = namespace;
+    this.description = description;
+    // Create 3 backingStores similar to default backing store
+  }
+
+  setTiers(tiers: Tier[]) {
+    this.tiers = tiers;
+  }
+
+  createBC = () => {
+    cy.log("General Data Page");
+    this.setGeneralData();
+    cy.log("Done setting General Data Page");
+    cy.contains('Next').click();
+    // Placement Policy Page
+    this.setPlacementPolicy();
+    cy.contains('Next').click();
+    // Backing Store Selection Page
+    this.setBackingStores();
+    cy.contains('Next').click();
+    // Review Page Data Extraction
+    cy.get('#content-scrollable > div.nb-create-bc-wizard > div > div > footer > button.pf-c-button.pf-m-primary').click();
```

Comment:
```suggestion
    cy.get('#content-scrollable > div.nb-create-bc-wizard > div > div > footer > button.pf-c-button.pf-m-primary').click();
```
please use data-test. Or reuse the common selector that is used for the primary button.

Reply:
Done